src/models/Offer/Repository.ts

```diff
@@ -92,7 +92,23 @@ export const OfferRepository = {
     }),
   findByUuid: async (uuid: string) => {
     const offer = await Offer.findByPk(uuid);
-    if (!offer) throw new OfferNotFound(uuid);
+    if (!offer) throw new OfferNotFoundError(uuid);
+
+    return offer;
+  },
+  findByUuidTargetedTo: async (uuid: string, applicant: Applicant) => {
```

Comment:
hubieras usado el metodo anterior y filtrabas con js pero bueno, da igual

Reply:
yo creo lo mismo, es mejor no tener metodos de más para hacer lo mismo. además la firma de la función es algo confusa, sería mucho más claro encontrar la oferta y tirar un error if not `offer.targetedTo(applicant)`